migrations/Version202211161003288041_taoAdvancedSearch.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoAdvancedSearch\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\generis\model\data\event\ResourceUpdated;
+use oat\oatbox\event\EventManager;
+use oat\oatbox\event\EventManagerAwareTrait;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoAdvancedSearch\model\Index\Listener\AgnosticEventListener;
+use oat\taoTests\models\event\TestUpdatedEvent;
+
+final class Version202211161003288041_taoAdvancedSearch extends AbstractMigration
+{
+    use EventManagerAwareTrait;
+
+    private const CALLBACK = [AgnosticEventListener::class, 'listen'];
+    private const HANDLED_EVENTS = [
+        ResourceUpdated::class,
+        TestUpdatedEvent::class,
+        QtiTestImportEvent::class,
```

Comment:
Shall we really handle those events now that we discovered that https://github.com/oat-sa/tao-core/pull/3673/files?diff=split&w=1#diff-6276791832b777d2ccdab8de777dce68467167617c591a79d9a135b788df71ebR96 is already doing it? 

Aren't we double processing?